views/ipv/page/pyi-triage-desktop-download-app.njk

```diff
@@ -1,68 +1,88 @@
-        <h2 class="govuk-heading-m">
-            {{ 'pages.pyiTriageDesktopDownload.section1.heading' | translate }}
-        </h2>
-    </div>
-    <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.section1.paragraph1' | translate }}</p>
-    <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.section1.paragraph2' | translate }}</p>
-
-    <p class="govuk-body">
-      <img
-        src='{{ qrCode }}'
-        alt="{{ 'pages.pyiTriageDesktopDownload.section1.qrCodeAlt' | translate }}"
-      />
-    </p>
-
-    {{ govukDetails({
-        summaryText: 'pages.pyiTriageDesktopDownload.section1.details1.summaryText' | translate,
-        text: 'pages.pyiTriageDesktopDownload.section1.details1.text' | translate
-    }) }}
-
-    {{ govukDetails({
-        summaryText: 'pages.pyiTriageDesktopDownload.section1.details2.summaryText' | translate,
-        text: 'pages.pyiTriageDesktopDownload.section1.details2.text' | translate
-    }) }}
-
-    {{ govukDetails({
-        summaryText: 'pages.pyiTriageDesktopDownload.section1.details3.summaryText' | translate,
-        text: 'pages.pyiTriageDesktopDownload.section1.details3.text' | translate
-    }) }}
-
-    {{ govukDetails({
-        summaryText: 'pages.pyiTriageDesktopDownload.section1.details4.summaryText' | translate,
-        html: 'pages.pyiTriageDesktopDownload.section1.details4.html' | translate
-    }) }}
-
-    {{ govukDetails({
-        summaryText: 'pages.pyiTriageDesktopDownload.section1.details5.summaryText' | translate,
-        text: 'pages.pyiTriageDesktopDownload.section1.details5.text' | translate
-    }) }}
-
-    <div class="circle-header-wrapper">
-        <p class="circle">{{ "pages.pyiTriageDesktopDownload.section2.circle" | translate }}</p>
-        <h2 class="govuk-heading-m">
-            {{ 'pages.pyiTriageDesktopDownload.section2.heading' | translate }}
-        </h2>
-    </div>
-
-    <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.section2.paragraph1' | translate }}</p>
-    <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.section2.paragraph2' | translate }}</p>
-    <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.section2.paragraph3' | translate( { currentUrl: req.originalUrl } ) | safe }}</p>
-
-    {{ govukInsetText({
-        html: 'pages.pyiTriageDesktopDownload.section2.inset' | translate( { contactUsUrl: contactUsUrl } ) | safe
-    }) }}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">
+      {{ 'pages.pyiTriageDesktopDownload.header' | translate }}
+  </h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.content.paragraph1' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pyiTriageDesktopDownload.content.paragraph2' | translate }}</p>
+  <ol class="govuk-list govuk-list--number">
+    <li class="govuk-heading-m">
+      {{ 'pages.pyiTriageDesktopDownload.content.subHeading1' | translate }}
+    </li>
+  </ol>
```

Comment:
Hmm it might be worth checking with UCD/accessibility but it feels a bit odd to just put the headings in a numbered list (and might be weird for screenreaders?) and it looks a bit off with the indentation - maybe these could just be headings?